packages/core/src/modules/dids/domain/DidResolver.ts

```diff
@@ -3,6 +3,8 @@ import type { ParsedDid, DidResolutionResult, DidResolutionOptions } from '../ty
 
 export interface DidResolver {
   readonly supportedMethods: string[]
+  readonly allowsCaching: boolean
```

Comment:
Since we are forced to add this flag in case we have a custom DID resolver, would this be a breaking change? It's not a problem but if that's the case should be added to the release notes.

Reply:
I can make it optional and set a default?

But might be good to make it explicit and a breaking change. Should be easy to configure.

What do you prefer?

Reply:
I think it is fine to make it explicit. We are also updating some other interfaces in this release as well (like the ones for AnonCreds registries), so it should not be a problem I guess.